### ISSUE TITLE
Compiled lumps differ in size from debotc output

### ISSUE THREAD

**Issue by ghost** [TarCV/botc#18]
I have talked to Torr about botc, he seemed happy about this and asked the following:
> Quick question: Should it be possible to exactly recreate the compiled bot scripts from zandronum.pk3 with debotc and botc? I tried decompiling and recompiling crashbot.lump, but the result differed from the original crashbot.lump.

**Comment by TarCV** [comment]
No, it should generate functionally same byte code, but byte by byte comparison would yield some differences. For example botc generates some sections in different order, and it does some simple optimizations like evaluating some expressions at compile time.

If it helps, I can finish disassemble feature in debotc, so that anyone interested can check that outputs are actually functionally same. Disasm output would be like:
```
00000010 DH_PUSHNUMBER(10)
00000018 DH_BOTCMD Delay
```

**Comment by ghost** [comment]
Maybe that will make him happy. In any case, I have linked him this issue, if he says anything I will post it.
